src/utils/variableType.js

```diff
@@ -7,10 +8,28 @@ import {isFunction} from './functionType';
  * @return {Boolean}
  */
 export function isUpdate(node) {
-  return node.type === 'UpdateExpression' && node.argument.type === 'Identifier';
+  return node.type === 'UpdateExpression';
 }
 
 /**
+ * True when node is an assignment with the variable on the LHS (like x= foo).
+ *
+ * @param {Object} node
+ * @param {Object} parent Immediate parent node (to determine context)
+ * @return {Boolean}
+ */
+export function isOnLHSAssignment(node, parent) {
```

Comment:
Naming things is one of the fundamentally hard problems in computer science. I guess my maths background means that `LHS` looks normal to me. I agree `isAssignedTo` is a much better name.